tests/e2e/parameters/conftest.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+from tests.e2e.parameters.infrastructure import ParametersStack
```

Comment:
I don't know what's going on, but I couldn't add the other commits with more e2e tests. I'm fixing it locally and will add it soon.